src/pages/apps/Read.tsx

```diff
@@ -67,446 +57,25 @@ export default function ReadApp() {
-                        <TableCell>Email</TableCell>
-                        <TableCell>Ending</TableCell>
-                      </TableRow>
-                    </TableHead>
-                    <TableBody>
-                      {Object.keys(groupMemberships(appGroup.active_user_memberships)).length > 0 ? (
-                        Object.entries(groupMemberships(appGroup.active_user_memberships))
-                          .sort(sortGroupMembers)
-                          .map(([userId, users]: [string, Array<OktaUserGroupMember>]) => (
-                            <TableRow key={userId}>
-                              <TableCell>
-                                <Link
-                                  to={`/users/${users[0].active_user?.email.toLowerCase()}`}
-                                  sx={{
-                                    textDecoration: 'none',
-                                    color: 'inherit',
-                                  }}
-                                  component={RouterLink}>
-                                  {displayUserName(users[0].active_user)}
-                                </Link>
-                              </TableCell>
-                              <TableCell>
-                                <Link
-                                  to={`/users/${users[0].active_user?.email.toLowerCase()}`}
-                                  sx={{
-                                    textDecoration: 'none',
-                                    color: 'inherit',
-                                  }}
-                                  component={RouterLink}>
-                                  {users[0].active_user?.email.toLowerCase()}
-                                </Link>
-                              </TableCell>
-                              <TableCell>
-                                <Ending memberships={users} />
-                              </TableCell>
-                            </TableRow>
-                          ))
-                      ) : (
-                        <TableRow>
-                          <TableCell>
-                            <Typography variant="body2" color="text.secondary">
-                              None
-                            </Typography>
-                          </TableCell>
-                        </TableRow>
-                      )}
-                    </TableBody>
-                    <TableFooter>
-                      <TableRow></TableRow>
-                    </TableFooter>
-                  </Table>
-                </TableContainer>
-              </Grid>
-            </React.Fragment>
-          ))}
+          <AppsHeader app={app} currentUser={currentUser} />
+          {app.active_owner_app_groups && (
+            <AppsAccordionListGroup
+              app_group={app.active_owner_app_groups}
+              list_group_title={'Owner Group'}
+              list_group_description={'Members can manage all app groups as implicit owners'}
```

Comment:
```suggestion
              list_group_description={'Owners can manage all app groups as implicit owners'}
```

Reply:
To be more clear, only members of the special built-in group `App-Access-Owners` are super-admins in Access. For a normal app in Access, owners of the associated `Owners` app group are admins for the app and all associated groups.